### PR TITLE
Patch Note

### PR DESCRIPTION
**Pull request by awakengithub**
- Added new class AdditionOfPowers
- Added new testclass AdditionOfPowersTesting

### CODE REVIEW

**Comment by awakenbad** [comment]
@konovchenko please have a look

File: src/main/java/com/company/QuadraticEquation.java [hunk 1]
```diff
@@ -3,13 +3,7 @@
 import java.util.Objects;
 import java.util.Scanner;
 
-/**
- * This code can help you to solve almost every Quadratic Equation task (probably).
- */
```
Comment (konovchenko) [inline src/main/java/com/company/QuadraticEquation.java:8]: Why is this deleted?

File: src/main/java/com/company/QuadraticEquation.java [hunk 2]
```diff
@@ -23,58 +17,69 @@ public static void main(String[] args) {
...
             }
         }
     }
 }
 
-/**
- This class has been used for returning your answers in the tests
- */
+
+class AdditionOfPowers {
```
Comment (konovchenko) [inline src/main/java/com/company/QuadraticEquation.java:58]: Please move to another file

**Comment by konovchenko** [comment]
Please apply requested changes

**Comment by konovchenko** [comment]
Looks good

**Review by konovchenko — APPROVED** [review]
